extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -9,11 +9,20 @@ $color-borders: #CECECE;
 
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
 
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/_colors.scss';
+
+$normal-spacing: 24px;
```

Comment:
Can we keep the same convention as in community page, i.e.
$content-spacing-.* ?